week04/code.js

```diff
@@ -0,0 +1,83 @@
+const btns = document.getElementById("btns");
+const start = document.getElementById("start");
+const reset = document.getElementById("reset");
+
+const human = document.getElementById("Hchoice");
+const com = document.getElementById("Cchoice");
+
+const score = document.getElementById("score");
+const result = document.getElementById("result");
+
+const elements = ["✌", "✊", "✋"];
+
+let myChoice = null;
+let Cchoice = null;
+
+let myScore = 0;
+let comScore = 0;
+
```

Comment:
timer를 함수 내부가 아닌, 파일 가장 바깥에서 선언하여 전역 변수에 저장하도록 합니다.

```suggestion
let timer;
```